src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+
+  --nav-item-height: 60px;
+  --nav-link-height: 15px;
+}
+
+.header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header__nav {
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-right: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  height: var(--nav-item-height);
+}
+
+.nav__item:last-child {
+  margin: 0;
+}
```

Comment:
better use tags for resetting margins and paddings, and u should combine them (u did it on the lesson)